cypress/integration/basic-functionalities/local-tests/compose/namespace/1_Create_a_namespace.spec.js

```diff
@@ -0,0 +1,75 @@
+/// <reference types="cypress" />
+const baseURL = Cypress.env('baseURL')
+const email = Cypress.env('user').email
+const password = Cypress.env('user').password
+
+// Before running this test first run the Server test 2 in order to create a user so you can log in.
```

Comment:
Let's rephrase this. Just specify that you have to signup. The folder structure makes it clear where the signup test is. No need to explicitly point it out.